modules/logging-release-notes-5-6-14.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+// logging-5-6-release-notes.adoc
+:_content-type: REFERENCE
+[id="logging-release-notes-5-6-14_{context}"]
+= Logging 5.6.14
+This release includes link:https://access.redhat.com/errata/RHSA-2023:7719[OpenShift Logging Bug Fix Release 5.6.14].
+
+[id="logging-release-notes-5-6-14-bug-fixes"]
+== Bug fixes
+
+* Before this update, the default alias was missing from the initial index in each log output while creating index patterns. As a result, the Kibana users were unable to create index patterns using {es-op}. With this update, adding the missing aliases to {es-op} resolves the issue. As a result, the Kibana users can create index patterns that include the `{app,infra,audit}-000001` indices. (link:https://issues.redhat.com/browse/LOG-4807[LOG-4807])
+
+* Before this update, the {loki-op} did not mount a custom CA Bundle to the ruler pods. As a result, it caused failing object storage access when evaluating alerting or recording rules. With this update, the {loki-op} mounts the custom CA bundle to all ruler pods. As a result, the ruler pods can download logs from object storage to evaluate alerting or recording rules. (link:https://issues.redhat.com/browse/LOG-4838[LOG-4838])
+
+[id="logging-release-notes-5-6-14-CVEs"]
+== CVEs
```

Comment:
I checked all of the links, and they all work. Nice job -- that was quite a list!